samples/bluetooth/peripheral_dis/prj.conf

```diff
@@ -1,5 +1,28 @@
 CONFIG_BT=y
 CONFIG_BT_PERIPHERAL=y
+
+
 CONFIG_BT_GATT_DIS=y
 CONFIG_BT_GATT_DIS_PNP=n
+CONFIG_BT_GATT_DIS_MODEL="Zephyr Model"
+CONFIG_BT_GATT_DIS_MANUF="Zephyr"
+CONFIG_BT_GATT_DIS_SERIAL_NUMBER=y
+CONFIG_BT_GATT_DIS_FW_REV=y
+CONFIG_BT_GATT_DIS_HW_REV=y
+CONFIG_BT_GATT_DIS_SW_REV=y
+CONFIG_BT_GATT_DIS_SERIAL_NUMBER_STR="Zephyr Serial"
+CONFIG_BT_GATT_DIS_FW_REV_STR="Zephyr Firmware"
+CONFIG_BT_GATT_DIS_HW_REV_STR="Zephyr Hardware"
+CONFIG_BT_GATT_DIS_SW_REV_STR="Zephyr Software"
+
 CONFIG_BT_DEVICE_NAME="DIS peripheral"
+
+# Below is setup to let DIS information be read from settings
+CONFIG_SETTINGS_RUNTIME=y
+CONFIG_SETTINGS_CUSTOM=y
+CONFIG_SETTINGS=y
+CONFIG_BT_SETTINGS=y
+
+CONFIG_BT_GATT_DIS_SETTINGS=y
+CONFIG_BT_GATT_DIS_STR_MAX=21
+
```

Comment:
OK

Reply:
Extra empty line at the end of the file, please remove